### DISCUSSION TITLE
middleware/elasticsearch/user-guide/migrate-es/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1878] [Announcements]
# middleware/elasticsearch/user-guide/migrate-es/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/middleware/elasticsearch/user-guide/migrate-es/

**Comment by liyuerich** [comment]
现在hwameistor提供UI进行pv的迁移，更方便。能否加入文档介绍？

**Reply by Michelle951** [reply]
相关文档已更新到文档站，见：https://docs.daocloud.io/middleware/elasticsearch/user-guide/migrate-es/
